powertools-logging/src/main/java/software/amazon/lambda/powertools/logging/internal/LambdaLoggingAspect.java

```diff
@@ -74,6 +76,12 @@ public final class LambdaLoggingAspect {
         }
 
         LEVEL_AT_INITIALISATION = LOG.getLevel();
+
+        if (System.getenv(EnvironmentVariables.LOGGER.LOG_EVENT) != null) {
```

Comment:
please, remain consistent with the other environment variables line 67/68, we'll create a proper file in v2...